src/cli/cli.cpp

```diff
@@ -5873,12 +5950,12 @@ template <> otError Interpreter::Process<Cmd("promiscuous")>(Arg aArgs[])
     return error;
 }
 
-void Interpreter::HandleLinkPcapReceive(const otRadioFrame *aFrame, bool aIsTx, void *aContext)
+void Cli::Interpreter::HandleLinkPcapReceive(const otRadioFrame *aFrame, bool aIsTx, void *aContext)
```

Comment:
Did you add this?

Reply:
not intentionally...just restored it to how it was before.  I must have inadvertently added that...thanks for catching!